lib/ansible/plugins/lookup/mongodb.py

```diff
@@ -202,7 +202,7 @@ def run(self, terms, variables, **kwargs):
 
   tasks:
     - debug: msg="Mongo has already started with the following PID [{{ item.pid }}] - full_data {{ item }} "
-      with_items:
+      loop:
```

Comment:
incorrect change, 'items' is doing an implicit flatten here

Reply:
It should be safe to replace "with_items" by "loop", but I didn't tested this case.
Feel free to close the PR if it is not valid case...
Thank you